cmake/alpakaCommon.cmake

```diff
@@ -152,6 +152,7 @@ endif()
 set(ALPAKA_OFFLOAD_MAX_BLOCK_SIZE "256" CACHE STRING "Maximum number threads per block to be suggested by any target offloading backends ANY_BT_OMP5 and ANY_BT_OACC.")
 option(ALPAKA_DEBUG_OFFLOAD_ASSUME_HOST "Allow host-only contructs like assert in offload code in debug mode." ON)
 set(ALPAKA_BLOCK_SHARED_DYN_MEMBER_ALLOC_KIB "47" CACHE STRING "Kibibytes (1024B) of memory to allocate for block shared memory for backends requiring static allocation (includes CPU_B_OMP2_T_SEQ, CPU_B_TBB_T_SEQ, CPU_B_SEQ_T_SEQ)")
+alpaka_compiler_option(RELOCATABLE_DEVICE_CODE "Enable relocatable device code for CUDA and HIP GPUs" OFF)
```

Comment:
should be `DEFAULT` and not `OFF`. We keep if possible the compiler default and let the user choose an explicit behavior.